Scripts/linux/dtlcommands.py

```diff
@@ -0,0 +1,359 @@
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
+#
+# MIT License
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import dtllabservice
+import json
+
+
+class AuthenticationCommands:
+    """Adds authorization arguments to the argument set.
+
+    Attributes:
+        None
+    """
+
+    def buildArguments(self, argParser):
+        """Constructs the command-line arguments used to support this command.
+
+        Args:
+            argParser (ArgumentParser) - the arguments parser used to parse the command-line.
+
+        Returns:
+            None
+        """
+        argParser.add_argument_group('Authentication')
+        argParser.add_argument('principal', help='Your Azure user name or Application/Client ID')
```

Comment:
Sometimes you write "user name" and sometimes "username". Minor but would be nice if this were consistent.